Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -2952,6 +2962,7 @@ public void restartGameSingle2()
         this.ShowHUDInfoCenter(string.Empty);
         this.DestroyAllExistingCloths();
         Application.LoadLevel(Application.loadedLevel);
+
```

Comment:
Remove this

Reply:
The whitespace?

Reply:
You added a random new line there, why? We don't need that, so remove it